BlogTemplate/Services/MarkdownRenderer.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Markdig;
+using Microsoft.AspNetCore.Html;
+
+namespace BlogTemplate._1.Services
+{
+    public class MarkdownRenderer
+    {
+        private static MarkdownPipeline pipeline = new MarkdownPipelineBuilder()
+               .UseDiagrams()
+               .UseAdvancedExtensions()
+               .UseYamlFrontMatter()
+               .DisableHtml()
+               .Build();
+        public HtmlString HtmlBody(string bodyText)

```

Comment:
I would name this something like RenderMarkdown or RenderMarkdownToHtml (this would be more appropriate if you could render it to something else as well).  The "Body" part only makes sense in the context that you're using it, not in the context of the service implementation itself.  And since it's a *Renderer, it makes sense to have a Render method.

Also, nit: leave an empty line before the method declaration.